### PR TITLE
Ammar admin job boards

### PR DESCRIPTION
**Pull request by ahaq0**
Add a job board to enable admin to view, search and delete all jobs created.

### CODE REVIEW

**Comment by arsidada** [comment]
@ahaq0 build is failing. likely because of an import that wasn’t committed to package.json

**Comment by ahmedhamodi** [comment]
In particular, the material-table component:
image

File: src/Components/Admin/JobPosts.js [hunk 1]
```diff
@@ -13,13 +93,81 @@ function withMyHook(Component){
...
+    }
+  }
+
+  fetchJobs = async () => {
+    const existingJobsData = await httpGet("jobs?status=ACTIVE,UNDER_REVIEW", localStorage.getItem("idToken"));
+    this.setState({
+      jobs: existingJobsData.data.jobs
+    })
+
+    console.log(existingJobsData);
```
Comment (ahmedhamodi) [inline src/Components/Admin/JobPosts.js:117]: Remove console log

File: src/Components/Admin/JobPosts.js [hunk 2]
```diff
@@ -13,13 +93,81 @@ function withMyHook(Component){
...
+
+  removeJob = async (jobID) => {
+
+    let removedJob = {
+      "job_status" : "REJECTED",
+    };
+    const removedJobsData = await httpPut(`jobs/${jobID}`, localStorage.getItem("idToken"), removedJob);
+    
+    console.log(removedJobsData);
+    // The job is already removed from state, so we just send the PUT request to remove it
```
Comment (ahmedhamodi) [inline src/Components/Admin/JobPosts.js:128]: Remove console log. Also, misplaced comment? Should be above the PUT request (or removed)

File: src/Components/Admin/JobPosts.js [hunk 3]
```diff
@@ -13,13 +93,81 @@ function withMyHook(Component){
...
+  }
+
+  componentDidMount() {
+    this.fetchJobs();
+    console.log(this.state);
```
Comment (ahmedhamodi) [inline src/Components/Admin/JobPosts.js:133]: Log!

File: src/Components/Admin/JobPosts.js [hunk 4]
```diff
@@ -13,13 +93,81 @@ function withMyHook(Component){
...
+    <MaterialTable
+      title="Global Job Postings"
+      columns={this.state.columns}
+      icons={tableIcons}
+      data={this.state.jobs}
+      options={{
+        paging:true,
+        pageSize:15,
+        emptyRowsWhenPaging:true,
+        pageSizeOptions:[5,10,15,30,50]
```
Comment (ahmedhamodi) [inline src/Components/Admin/JobPosts.js:148]: I'd be a fan to include an export button. In material table you can do:
`exportButton: true,
exportAllData: true`

**Review by ahmedhamodi — COMMENTED** [review]
A few comments! Overall, looks good. I'll do a more detailed test (ie. run the app and test) later.

**Review by arsidada — CHANGES_REQUESTED** [review]
fix import change

File: src/Components/Admin/JobPosts.js (hunk 1 above)
Comment (ahaq0) [reply src/Components/Admin/JobPosts.js:117]: oops sorry was too excited after we figured out the issue, I didn't look it over prior to making the PR.

File: src/Components/Admin/JobPosts.js [hunk 5]
```diff
@@ -13,13 +93,78 @@ function withMyHook(Component){
...
+      columns={this.state.columns}
+      icons={tableIcons}
+      data={this.state.jobs}
+      options={{
+        paging:true,
+        pageSize:15,
+        emptyRowsWhenPaging:true,
+        pageSizeOptions:[5,10,15,30,50],
+        exportTrue: true,
+        exportAllData: true
```
Comment (ahmedhamodi) [inline src/Components/Admin/JobPosts.js:145]: Should be: exportButton: true, exportAllData: true
not exportTrue

**Comment by ahmedhamodi** [comment]
Not sure why the build is failing. Does it run locally for you @ahaq0 ?

**Review by arsidada — APPROVED** [review]

**Review by ahmedhamodi — APPROVED** [review]
Yay it passed!!

**Pull request merged**